File: OLSource1/v1/VS_officedev/codesnippet/CSharp/how-to--programmatically-restore-selections-after-searches_4.cs
        internal void ReplaceRestoreSelection() 
        { 
            Word.Range start = Application.Selection.Range; 
            Word.Range searchArea = Application.ActiveDocument.Range(ref missing, ref missing); 

            searchArea.Find.ClearFormatting(); 
            searchArea.Find.Text = "find me"; 

            searchArea.Find.Replacement.ClearFormatting(); 
            searchArea.Find.Replacement.Text = "Found"; 

            object replaceAll = Word.WdReplace.wdReplaceAll; 

            searchArea.Find.Execute(
                ref missing, ref missing, ref missing, ref missing, ref missing,
                ref missing, ref missing, ref missing, ref missing, ref missing,
                ref replaceAll, ref missing, ref missing, ref missing, ref missing);

            start.Select(); 
        }

File: OLSource1/v1/VS_officedev/codesnippet/VisualBasic/how-to--programmatically-restore-selections-after-searches_4.vb
    Friend Sub ReplaceRestoreSelection()
        Dim start As Word.Range = Application.Selection.Range
        Dim searchArea As Word.Range = Application.ActiveDocument.Range

        searchArea.Find.ClearFormatting()
        searchArea.Find.Text = "find me"

        searchArea.Find.Replacement.ClearFormatting()
        searchArea.Find.Replacement.Text = "Found"

        searchArea.Find.Execute(Replace:=Word.WdReplace.wdReplaceAll)

        start.Select()
    End Sub